array_analyzer/extract/image_parser.py

```diff
@@ -117,6 +117,29 @@ def thresh_and_binarize(image_, method='rosin', invert=True):
     return spots
 
 
+def get_well_mask(image_, segmethod='rosin'):
+
+    well_mask = ~thresh_and_binarize(image_, method=segmethod, invert=True)
```

Comment:
```
~thresh_and_binarize(image_, method=segmethod, invert=True)
```
is equivalent to
```
thresh_and_binarize(image_, method=segmethod, invert=False)
```

Here I would prefer to use invert=False if only because it is already an argument to the function

Reply:
Fixed